bin/metrics-nrpe.rb

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env ruby
+#
+#   NRPE Metrics
+#
+# DESCRIPTION:
+#   This is a simple script to collect metrics from an NRPE plugin
+#
+# OUTPUT:
+#   metric data
+#
+# DEPENDENCIES:
+#   gem: sensu-plugin
+#   gem: nrpeclient
+#
+# USAGE:
+#   check-nrpe -H host -c check_plugin -a 'plugin args' -p prefix -s suffix
+#
+# NOTES:
+#   regex from https://github.com/naparuba/shinken/blob/master/shinken/misc/perfdata.py
+#
+# LICENSE:
+#   Copyright (c) 2016 Scott Saunders <scosist@gmail.com>
+#   Based on metrics-snmp.rb by Double Negative Limited
+#
+# Released under the same terms as Sensu (the MIT license); see LICENSE
+# for details.
+
+require 'sensu-plugin/metric/cli'
+require 'nrpeclient'
+
+# Class that collects and outputs NRPE metrics in graphite format
+class NRPEGraphite < Sensu::Plugin::Metric::CLI::Graphite
+  option :host,
+         short: '-H host',
+         boolean: true,
+         default: '127.0.0.1',
+         required: true
+
+  option :check,
+         short: '-c check_plugin',
+         boolean: true,
+         default: '',
+         required: true
+
+  option :args,
+         short: '-a args',
+         default: ''
+
+  option :prefix,
+         short: '-p prefix',
+         description: 'prefix to attach to graphite path'
+
+  option :suffix,
+         short: '-s suffix',
+         description: 'suffix to attach to graphite path',
+         required: true
+
+  option :port,
+         short: '-P port',
+         description: 'port to use (default:5666)',
+         default: '5666'
+
+  option :ssl,
```

Comment:
same comment, we should have an option to disable not an option to enable it as that should be the recommended path.